python/packages/autogen-agentchat/src/autogen_agentchat/agents/_assistant_agent.py

```diff
@@ -292,7 +330,11 @@ async def on_messages_stream(
                     handoffs.append(self._handoffs[call.name])
             if len(handoffs) > 0:
                 if len(handoffs) > 1:
-                    raise ValueError(f"Multiple handoffs detected: {[handoff.name for handoff in handoffs]}")
+                    # show warning if multiple handoffs detected
```

Comment:
We should be disabling parallel tool calling if handoff is present.

Reply:
I am not too familiar with handoffs so maybe we keep them to a different PR?